src/components/Details.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-unused-expressions */
+import { useSelector } from 'react-redux';
```

Comment:
- [ ] The linters are there to enforce code quality and should not be disabled. Kindly enable the linters and fix the issues raised.